docs/clustering.md

```diff
@@ -163,13 +163,39 @@ performant, but is not highly available.
 The separate server for the database and redis also increases performance.
 
 This relies on a round-robin DNS record that points to all 2 Pulp servers:
+
 * `example-pulp-load-balanced-hostname.fqdn`
 
 This also relies on shared storage (such as NFS) being hosted at the following address:
+
 * `example-nfs-server:/var/lib/pulp`
 
 Adjust the pre_task `Mount /var/lib/pulp` as needed for your shared storage.
 
+Install the `nfs-utils` package:
+
+```
+# yum install nfs-utils
+```
+Mount the NFS share:
+
+* use the `showmount` utility to discover the shares from NFS server:
+```
+$ showmount --exports <my-nfs-server-address>
+```
+* mount the `/var/lib/pulp` and `/var/lib/pulp/pulpcore_static` [share points](https://github.com/pulp/pulp_installer/blob/main/roles/pulp_webserver/README.md#role-variables):
+```
+# mount <my-nfs-server-address>:<exported-file-system-for-pulp-user-home> /var/lib/pulp
+# mount <my-nfs-server-address>:<exported-file-system-for-webserver-static-dir> /var/lib/pulp/pulpcore_static
+```
+
+For servers running with SELinux in Enforcing mode, the following SELinux file contexts are needed for correct operation:
+
+|**Directory** | **SELinux Context**
+|--------- | ---------------
+|`/var/lib/pulp` | system_u:object_r:pulpcore_etc_t:s0
```

Comment:
system_u:object_r:pulpcore_var_lib_t:s0

Also, the selinux label is not available until after pulp is installed, I am making it remount mid-install with it:
https://github.com/pulp/pulp_installer/issues/1196